packages/aws-cdk/lib/api/aws-auth/account-cache.ts

```diff
@@ -16,13 +15,24 @@ export class AccountAccessKeyCache {
    */
   public static readonly MAX_ENTRIES = 1000;
 
+  /**
+   * The default path used for the accounts access key cache
+   */
+  public static get DEFAULT_PATH(): string {
+    // needs to be a getter because cdkCacheDir can be set via env variable and might change
```

Comment:
...during tests?

Reply:
during tests, but in `toolkit-lib` it could be at any point of the code execution.